package.json

```diff
@@ -20,7 +20,7 @@
     "cross-env": "^5.1.3",
     "express": "^4.16.2",
     "jscs": "^3.0.7",
-    "jshint": "^2.9.5",
+    "jshint": "^0.8.0",
```

Comment:
Is this necessary?

Reply:
I see that you said:

> Updated jshint version to remove optional phantomjs dependency.

Seems odd that we'd have to go back 2 major versions just to remove PhantomJS.

Reply:
jscs should be dropped in favor of eslint.  probably should just drop jshint too in favor of eslint.

Reply:
As I mentioned earlier I did this change only to remove opional phantomjs dependency, but I agree with you that instead of dropping jshint version we should drop jshint and jscs from the  package.json.
I will be happy to update this PR accordingly if you agree.